cpu/cortexm_common/panic.c

```diff
@@ -32,3 +40,14 @@ void panic_arch(void)
     }
 #endif
 }
+
+#ifdef DEVELHELP
+static void print_IPSR(void)
+{
+    uint8_t ipsr = (uint8_t)(__get_IPSR() & 0xFF);
+    if(ipsr) {
+        LOG_ERROR("Trapped interrupt number: %d\n", ((int)ipsr) - 16);
+        LOG_ERROR("Have a look at IRQn_Type to see which interrupt this is\n");
```

Comment:
The output string is really verbose (and hence memory consuming). Can you strip it down to the minimum, please?